### DISCUSSION TITLE
My NFT doesn't show up on collectable

### DISCUSSION THREAD

**Discussion by yunzin** [phantom/docs#273] [Help & Support] [answered]
### Summary

![image](https://github.com/user-attachments/assets/0a68ce2c-1dca-4ff0-9729-9c2551996b7b)
![image](https://github.com/user-attachments/assets/f076a09b-3ef7-4370-84da-09e4b1026d2c)

### Steps to Reproduce

![image](https://github.com/user-attachments/assets/0a68ce2c-1dca-4ff0-9729-9c2551996b7b)
![image](https://github.com/user-attachments/assets/f076a09b-3ef7-4370-84da-09e4b1026d2c)

### Is there an existing discussion for this?

- [X] I have searched the existing discussions

**Comment by adamdelphantom** [comment]
Hello @yunzin,

Would you mind providing the contract address for this nft, please?

**Reply by yunzin** [reply]
Hi @adamdelphantom 
the contract address is

- 0x00c66eec8fa56d9195fd5ca5333956ef1ae3c858

**Reply by adamdelphantom** [reply, marked as answer]
@yunzin this should show up now
